github-actions/utils/setup-bazel-remote-exec/action.yml

```diff
@@ -3,6 +3,7 @@ description: 'Sets up Bazel remote execution.'
 author: 'Angular'
 inputs:
   bazelrc:
+    default: '.bazelrc'
```

Comment:
Is this right? Setting up the remote bazelrc was made intentionally opt-in because it might be different per setup. I think it was components repo or something.

Also IIRC we used the bazelrc user for this because we don't want to cause Git changes